Skia-binding fails with textlayout feature

Without the `textlayout` feature I don't have any issues building, but as soon as I add it I get the following error:

```
error[E0412]: cannot find type `std__Rb_tree_color` in this scope
   --> /.../target/debug/build/skia-bindings-30bbc81fb1d2ba99/out/skia/bindings.rs:926:19
    |
926 |     pub _M_color: std__Rb_tree_color,
    |                   ^^^^^^^^^^^^^^^^^^ not found in this scope

For more information about this error, try `rustc --explain E0412`.
error: could not compile `skia-bindings` (lib) due to 1 previous error
```

This error can be avoided by adding the `all-linux` feature, but any modification to it (except removing gl/egl, since they're a Wayland prerequisite) will cause this error to pop up again. Just `textlayout` on its own with default features also seems to work, but as soon as `egl` is added or default features are removed it seems to break.

--- a/docker/linux/archlinux-amd64/Dockerfile
+++ b/docker/linux/archlinux-amd64/Dockerfile
@@ -1,0 +1,24 @@
+FROM --platform=linux/amd64 archlinux:latest
+
+ENV RUSTFLAGS="-C target-feature=-crt-static" \
+    CC="clang" \
+    CXX="clang++" \
+    GN_EXE="gn" \
+    SKIA_GN_COMMAND="/usr/bin/gn" \
+    SKIA_NINJA_COMMAND="/usr/bin/ninja"
+
+RUN pacman -Syu --noconfirm \
+    curl git python clang llvm \
+    musl openssl gcc make \
+    fontconfig ttf-dejavu \
+    gn ninja
+
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --profile minimal
+ENV PATH="/root/.cargo/bin:${PATH}"
+
+# WORKDIR /code
+
+# COPY . /code/rust-skia/
+
+# RUN cd rust-skia && \
+#     cargo build -vv -p skia-safe --features=embed-freetype,textlayout

--- a/skia-bindings/build_support/skia_bindgen.rs
+++ b/skia-bindings/build_support/skia_bindgen.rs
@@ -481,6 +481,8 @@ const BLOCKLISTED_TYPES: &[&str] = &[
     "SkRuntimeEffect_ConstIterable.*",
     // Linux LLVM9 c++17
     "std::_Rb_tree.*",
+    // archlinux
+    "std::__rb_tree.*",
     // Linux LLVM9 c++17 with SKIA_DEBUG=1
     "std::__cxx.*",
     "std::array.*",

